stdlib/2and3/hmac.pyi

```diff
@@ -33,15 +33,16 @@ class HMAC:
     if sys.version_info >= (3, 4):
         block_size: int
         name: str
+    def __init__(self, key: _B, msg: Optional[ReadableBuffer] = ..., digestmod: _DigestMod = ...) -> None: ...
```

Comment:
Just use `bytes` for the `key` argument. It's an alias for `Union[bytes, bytearray, memoryview]` in argument types anyway. I know that other arguments are also annotated with `_B`, which should be fixed at some point.

Reply:
Can you link me to why bytes is an alias for that? Don't see mention of this in PEP 484.

Reply:
It's mentioned in the documentation for [typing.ByteString](https://docs.python.org/3/library/typing.html#typing.ByteString).